src/color-picker/color-picker.scss

```diff
@@ -12,6 +12,9 @@ $iui-active-box-shadow: 0 0 0 $iui-xxs rgba(t(iui-color-foreground-primary-rgb),
 $iui-hover-box-shadow: 0 0 1px $iui-xxs + 1 rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-5));
 
 @mixin iui-color-picker {
+  --iui-color-field-hue: initial;
```

Comment:
I assume should still have `iui-color-picker` prefix, cause is a part of `ColorPicker` component.

Reply:
ok changed

Reply:
Well, I went with the name of the mixin that is initializing these variables. For example, it doesn't make sense to use `iui-color-picker-background` instead of `iui-color-swatch-background`.

Reply:
i was talking more like `iui-color-picker-swatch-background`, but ok. Minor